test/specs/elements/Label/Label-test.js

```diff
@@ -45,17 +45,6 @@ describe('Label', () => {
   })
 
   describe('removeIcon', () => {
-    it('has no icon without onRemove', () => {
-      shallow(<Label />)
-        .should.not.have.descendants('Icon')
-    })
-
-    it('has delete icon by default', () => {
```

Comment:
I believe this test would still apply since there is a default icon, it is just not applied in `defaultProps`, true?